dockers/docker-base-stretch/Dockerfile.j2

```diff
@@ -53,7 +53,10 @@ RUN apt-get update &&        \
         libjemalloc1         \
         liblua5.1-0          \
         lua-bitop            \
-        lua-cjson
+        lua-cjson            \
+# For DHCP Monitor tool
+        libexplain51         \
+        libevent-2.0-5
```

Comment:
Is it necessary to explicitly install the runtime dependencies here? If you specify the dependencies in your Debian package config, our build system should implicitly install them when installing this package in the pertinent container image. We would like to move away from explicitly installing runtime dependencies as much as possible, as it means there is more to keep track of and maintain.

Reply:
Just checked and explicit install is not required. removed those lines